app/assets/javascripts/required_fields.coffee

```diff
@@ -70,6 +70,10 @@ $(document).ready ->
     if $fields.length > 0
       # get any values from the fields at the current data-level
       fieldsWithValues = $fields.filter ->
+        # TODO: I wanted to change this filtering to use jQuery to be more
```

Comment:
@ryanmiller-1 , @jdf-eos , just wanted to make you aware that I had to revert the change here and added comments, in case that affects your approval. There was bad behavior/a failing test I was not able to resolve quickly and didn't want to hold up the ticket.